aten/src/ATen/native/TensorShape.cpp

```diff
@@ -61,7 +61,9 @@ Tensor& set_storage_cpu_(Tensor& result, Storage storage, int64_t storage_offset
   result.unsafeGetTensorImpl()->set_storage_offset(storage_offset);
   c10::optional<IntArrayRef> stride_opt = stride.data() != nullptr ?
                                           c10::optional<IntArrayRef>(stride) : c10::nullopt;
-  at::native::resize_impl_cpu_(result.unsafeGetTensorImpl(), size, stride_opt);
+  /* We can re-use this kernel for the meta device.\
```

Comment:
I was getting a compiler warning locally when I used `//`, and I wasn't sure if that would fail CI too. But it probably shouldn't, I'll just change it back.